### DISCUSSION TITLE
Any way to force stop the calculation?

### DISCUSSION THREAD

**Discussion by d07RiV** [AbdBarho/stable-diffusion-webui-docker#351] [Q&A]
Sometimes the process gets stuck at 90% or so and makes my PC unusable for several minutes. Stopping the docker container doesn't do anything. On top of that vmmem process remains and uses 30 gigs of ram and can't be terminated. Is there anything I can do about it other than a reboot?

**Comment by AbdBarho** [comment]
this is a problem with docker on windows, because it is all hacked through WSL.

You can try quitting docker desktop and running `wsl --shutdown` in powershell
